chef/cookbooks/network/recipes/default.rb

```diff
@@ -301,7 +301,7 @@ def kill_nic(nic)
     net_ifs << our_iface.name
   end
   if network["mtu"]
-    if name == "admin" or name == "storage"
+    if %w(admin, storage, os_sdn).include? name
```

Comment:
Is a regular array syntax preferred? Coming from Python it looks more 'normal' to me too :) But we are using the `%w(` syntax over 30 times in this repository and over 300 times in crowbar-openstack, which is why I figured Rubyists prefer it that way.